src/webob/acceptparse.py

```diff
@@ -901,25 +699,35 @@ def acceptable_offers(self, offers):
         """
         Return the offers that are acceptable according to the header.
 
-        The offers are returned in descending order of preference, where
-        preference is indicated by the qvalue of the media range in the header
-        that best matches the offer.
+        The matched offers are returned in descending order of preference,
+        where preference is indicated by the qvalue of the media range in the
+        header that best matches the offer.
 
         This uses the matching rules described in :rfc:`RFC 7231, section 5.3.2
         <7231#section-5.3.2>`.
 
-        Any offers that cannot be parsed via
-        :meth:`.Accept.parse_offer` will be ignored.
-
-        :param offers: ``iterable`` of ``str`` media types (media types can
-                       include media type parameters) or pre-parsed instances
-                       of :class:`.AcceptOffer`.
-        :return: A list of tuples of the form (media type, qvalue), in
-                 descending order of qvalue. Where two offers have the same
-                 qvalue, they are returned in the same order as their order in
-                 `offers`.
-        """
-        parsed = self.parsed
+        Any offers that cannot be parsed via :meth:`.parse_offer`
+        will be ignored.
+
+        :param offers:
+            ``iterable`` of ``str`` media types (media types can
+            include media type parameters) or pre-parsed instances
+            of :class:`.AcceptOffer`.
+        :return:
+            A list of tuples of the form (media type, qvalue), in
+            descending order of qvalue. Where two offers match the same
+            qvalue, they are returned in the same order as their order in
+            ``offers``.
```

Comment:
The documentation for the return value from `_AcceptInvalidOrNoHeader.acceptable_offers` needs to be included here to explain what happens with the return value when the header is invalid or missing?